tests/integration/authy/v1/service/test_entity.py

```diff
@@ -19,10 +19,16 @@ def test_create_request(self):
 
         with self.assertRaises(TwilioException):
             self.client.authy.v1.services(sid="ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX") \
-                                .entities.create(identity="identity")
+                                .entities.create(identity="identity", twilio_authy_sandbox_mode="twilio_authy_sandbox_mode")
 
         values = {'Identity': "identity", }
 
+        headers = {'Twilio-Authy-Sandbox-Mode': "twilio_authy_sandbox_mode", }
+        self.holodeck.assert_has_request(Request(
+            'post',
+            'https://authy.twilio.com/v1/Services/ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Entities',
+            headers=headers,
+        ))
```

Comment:
Why is a second assertion added?

Reply:
Easier logic to add to the integration test generator than rewriting the existing assert to include headers. It matches the Node helper lib header integration tests